website/docs/r/is_share_mount_target.html.markdown

```diff
@@ -21,30 +21,107 @@ This is a Beta feature and it is subject to change in the GA release
 ## Example Usage
 
 ```hcl
-resource "ibm_is_vpc" "vpc" {
+resource "ibm_is_vpc" "example" {
   name = "my-vpc"
 }
 
-resource "ibm_is_share" "is_share" {
+resource "ibm_is_share" "example" {
   name = "my-share"
   size = 200
   profile = "tier-3iops"
   zone = "us-south-2"
 }
 
-resource "ibm_is_share_mount_target" "is_share_target" {
+resource "ibm_is_share_mount_target" "example" {
   share = ibm_is_share.is_share.id
   vpc = ibm_is_vpc.vpc.id
   name = "my-share-target"
 }`
 ```
+```
+//Create mount target with virtual network interface that has primary ip name and subnet id
+resource "ibm_is_vpc" "example" {
+  name = "my-vpc"
+}
 
+resource "ibm_is_share" "example" {
+  access_control_mode = "security_group"
+  zone    = "br-sao-2"
+  size    = 9600
+  name    = "my-example-share1"
+  profile = "dp2"
```

Comment:
@deepaksibm Can we add a note on top of this document that moving forward, the `tier-3iops` profile will not longer be supported. Supported profile is `dp2` @hkantare It would be helpful to add this to release notes please. thanks.

Reply:
added noted

Reply:
added note in share resource doc